phnt/include/ntpsapi.h

```diff
@@ -1414,7 +1414,7 @@ typedef struct _THREAD_UMS_INFORMATION
  */
 typedef struct _THREAD_NAME_INFORMATION
 {
-    // A Unicode string that specifies the description of the thread.
+    // An Unicode string that specifies the description of the thread.
```

Comment:
"A Unicode" is correct, "An Unicode" isn't, refs:
https://en.wikipedia.org/wiki/Unicode
https://en.wikipedia.org/wiki/Unicode_block
https://www.unicode.org/glossary/

Reply:
Thank you, I gonna revert this.

Reply:
I fixed it as part of https://github.com/winsiderss/systeminformer/pull/2316.